### ISSUE TITLE
title redundant in last_post_message

### ISSUE THREAD

**Issue by Zatalyz** [SimpleMachines/SMF#7621]
#### Description
I use the Wave extension to try to improve the accessibility of my forum. It came to my attention that some links had a redundant "title" element (not really a accessibility bug, but open a warning). Indeed, title is equal to link text in this kind of case. This concerns in particular what is called by the following code in the themes:

`$board['last_post']['last_post_message']`

The modification impacts the Source part, it seems to me that this can be classified as a bug, rather than as a theme hack. 

I'm not sure (I'm not a dev, I barely understand php), but it seems to me that line 417 of Source/Subs-BoardIndex.php is the offending part:

`$this_last_post['link'] = '<a href="' . $this_last_post['href'] . '" title=" test' . $row_board['subject'] . '">' . $row_board['short_subject'] . '</a>';`

Replace with 
`$this_last_post['link'] = '<a href="' . $this_last_post['href'] . '">' . $row_board['short_subject'] . '</a>';`

This solves the problem but I am unable to assess the side effects. Maybe there are a really good reason to duplicate text link in the title link ?

### Steps to reproduce
1. Open a forum with the default theme
2. Inspect link to "last post" on a board. There are title on the link, redundant with text link.

### Environment
- Version/Git revision: SMF 2.1.2

**Comment by jdarwood007** [comment]
Can you provide documentation showing why this is wrong?

I myself use the lighthouse tool in Chrome to test things like this and while we don't have a perfect score, there doesn't seem to be a complaint about the title attribute matching the inner of the tag.

**Comment by DiegoAndresCortes** [comment]
From what I could find, screen readers skip it so they are not useful, but they are not harmful at all.

**Comment by Zatalyz** [comment]
I don't know if it's "wrong". Redundant, sure, and it seem that it only brings noise.  I don't see a reason to use "title" here, because we have no more information about the link, but if _Wave_ didn't lighted it, I probably never notice that :)

In any case, this is not a major problem and I report this only with the hope it can improve the beauty of code.  It's also possible that it has a real use and in that case, I would have learned something. I'm a beginner and I wouldn't pretend that something "must" be done. If you consider that this titles are useful, the ticket can be closed :)
